lib/bundles/inspec-compliance/api.rb

```diff
@@ -151,12 +178,23 @@ def self.get_token(config)
 
     def self.target_url(config, profile)
       if config['server_type'] == 'automate'
-        target = "#{config['server']}/#{profile}/tar"
+        owner, id = profile.split('/')
+        target = "#{config['server']}/profiles/#{owner}/#{id}/tar"
       else
         owner, id = profile.split('/')
         target = "#{config['server']}/owners/#{owner}/compliance/#{id}/tar"
       end
       target
     end
+
+    # returns a parsed url for `admin/profile` or `compliance://admin/profile`
+    def self.sanitize_profile_name(profile)
```

Comment:
I'm sure I'm missing something with this method... The end state is to return `'admin/profile'` regardless of whether it was supplied with a URI scheme or not, correct? It seems like we don't need to prepend a non-schemed URI with a scheme just to remove the scheme from the string. In fact, that's kinda what was being done in an earlier version of this method is removed later in this PR. 

Could this method be simplified to:

```
uri = URI(profile)
"#{uri.host}#{uri.path}"
```


```
irb(main):021:0* a = URI('compliance://admin/profile')
=> #<URI::Generic:0x007fc51b0b1980 URL:compliance://admin/profile>
irb(main):022:0> b = URI('admin/profile')
=> #<URI::Generic:0x007fc51b0a8420 URL:admin/profile>
irb(main):023:0> "#{a.host}#{a.path}"
=> "admin/profile"
irb(main):024:0> "#{b.host}#{b.path}"
=> "admin/profile"
```

Like I said, I'm sure I'm missing something here so feel free to educate me!